scripts/acn/run_acn_node_standalone.py

```diff
@@ -207,7 +216,23 @@ def run(self):
 
         cmd = [self.binary, config_file]
 
-        self._proc = subprocess.Popen(cmd, shell=False,)  # nosec
+        if self.config.config[AcnNodeConfig.ACN_LOG_FILE] != "":
```

Comment:
This is to print both to stdout and to log file, so that `kubectl get logs` will still usable